### DISCUSSION TITLE
Github not able to identify JavaScript files

### DISCUSSION THREAD

**Discussion by vishalsingh2972** [community/community#60346] [Repositories]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Almost 90% of my code is JavaScript but GitHub for this repo shows something like this 
**Output I am getting-**
![image](https://github.com/vishalsingh2972/JS-Chess/assets/106817047/73885c8f-cb23-49a6-a430-0736224b715e)

**Expected Output-**
![image](https://github.com/vishalsingh2972/JS-Chess/assets/106817047/a1faadd8-6931-4818-8416-b32474e57d59)

Here is the GitHub Repo: https://github.com/vishalsingh2972/JS-Chess

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by eneshazr** [comment]
There may be issues with the cache. Try making a change in your repository.

**Reply by vishalsingh2972** [reply]
tried , can you submit a PR for the issue if possible!

**Comment by ronakjkothari** [comment]
My guess is that GitHub is picking up due to the name and content of the javascript files that it is boilerplate (generated) code, and GitHub will ignore that. I will leave some documentation that you could feel free to check out regarding the language bar for GitHub.

- [About repository languages](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-repository-languages)
- [Troubleshooting Linguist](https://github.com/github-linguist/linguist/blob/master/docs/troubleshooting.md)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @vishalsingh2972 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by vishalsingh2972** [reply]
That's so kind of you @queenofcorgis , thank you very much :)
